unit-tests/resources/commcare-apps/flipper/profile.ccpr

```diff
@@ -0,0 +1,106 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<profile version="17"
+         update="https://www.commcarehq.org/a/flipper/apps/download/05938664cd52392fb39a3567d8d8902a/media_profile.ccpr?latest=true"
+         requiredMajor="2"
+         requiredMinor="22"
+         uniqueid="36c0bdd028d14a52cbff95bb1bfd0962"
+         name="Untitled Application"
+         descriptor="Profile File">
+    <property key="BackupMode" value="file_mode"/>
+    <property key="backup-url" value="file:///E:/CommCare.Backup"/>
+    <property key="restore-url" value="file:///E:/CommCare.Backup"/>
+    <property key="ota-restore-url" value="https://www.commcarehq.org/a/flipper/phone/restore/"/>
```

Comment:
Can we minimize these app resources a bit?

If you look at the MockApp app resources they are stripped super bare, which I'd prefer we do for these setups. Using locale files and keeping around a bunch of config properties seems like it's likely to cause confusion and not be clear on what the tests are doing.